### DISCUSSION TITLE
Support Twitch's new tool for helping prevent ban evasion

### DISCUSSION THREAD

**Discussion by ALazyMeme** [Chatterino/chatterino2#3380] [Ideas] [answered]
I'm not sure how it'll be implemented, but chat ban evasion flagging is a new thing for webchat. I'd love to see something similar added to highlights in chatterino: https://safety.twitch.tv/s/safety-news/detail?post=Using-Machine-Learning-to-Curb-Ban-Evasion

**Comment by Felanbird** [comment]
`low-trust-users.<user-id>.<channel-id>` pubsub topic

**Reply by karl-police** [reply]
> `low-trust-users.<user-id>.<channel-id>` pubsub topic

there are a few other pubsub topics as well, I think

**Reply by iProdigy** [reply]
This is now official, and can be used in c2: https://www.twitch.tv/videos/1604648673?t=00h20m45s

We can:
* log when a user's treatment status changes (e.g., monitored, restricted, none)
* read restricted messages that aren't sent over irc

Before the official docs are live, the fields can be seen [here](https://github.com/twitch4j/twitch4j/pull/491/files#diff-c0e5497011dddbe9772b61632ac75ee4e1660488c70fcf38707856765e238d2e)

**Comment by karl-police** [comment]
also someone talked about this https://dev.twitch.tv/docs/eventsub

**Comment by iProdigy** [comment, marked as answer]
Completed in #5056 to the current extent possible (no helix endpoints available for suspicious user management)
